content/en/blog/_posts/2022-04-15-Increasing-the-security-bar-in-Ingress-NGINX/index.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog
+title: 'Increasing the security bar in Ingress-NGINX'
+date: 2022-04-15
+slug: ingress-nginx-security-release
+---
+
+**Authors:** Ricardo Katz (VMware), James Strong (Chainguard)
+
+Ingress may be one of the most targeted components of Kubernetes. In fact, it usually is an HTTP Proxy, exposed to the Internet, containing multiple websites, and with some privileged access to Kubernetes API (mainly secret access for TLS certificates).
+
+While it is a risky component in your architecture, it is still required to properly expose your services.
```

Comment:
You don't need an Ingress to expose services (you could use Service or Gateway, for example). Maybe reword “required” to “the most popular way”.